tools/build_cdn.js

```diff
@@ -58,6 +64,22 @@ async function buildCDN(options) {
   log("-----");
 }
 
+
+async function buildSRIDigests(shas) {
+  const temp = await fs.readFile("./tools/templates/DIGESTS.md")
+  const DIGEST_MD = temp.toString()
+
+  const version = require("../package").version
+  const digestList = Object.entries(shas).map((k,v) => `${v} ${k}`).join("\n")
```

Comment:
```suggestion
  const digestList = Object.entries(shas).map(([k,v]) => `${v} ${k}`).join("\n")
```

I think we want this to be destructured because otherwise, it'll appear as:

```
1 highlight.js,sha384-gAbqRnGhyx7nwFTx5JRXNF0zpofOgWMethd/B4DgtBSvr66L8CN8/4I483QS5G0/
```

Reply:
I'll take another look at this.